demo.md

```diff
@@ -0,0 +1,24 @@
+## Demo script:
+
+- ERC-4337 compatible Smart Contract wallet
+- Unique feature: gasless off-chain txs with instant finality enabled by HTLC technology, as used in Bitcoin Lightning
+- Setup required:
+  - wallet owners must designate an Intermediary in their SC wallet
+  - wallet owners and Intermediary must sign-off on some amount of wallet funds designated for off-chain txs
+  - run in-browser client to interact with SC wallet and peers
+  - `Explain browser tab setup with Alice(left), Irene(middle), Bob(right)`
+- How it works:
+  - on-chain txs are 2/2 multi-sig (user + intermediary)
+  - `Show on-chain tx from Alice to Metamask account`
+  - user can remove intermediary at any point to claim full ownership of wallet/funds
+  - `Point to 'Eject' button on UI`
+  - off-chain payments are multisig between payor, intermediary, and payee
```

Comment:
I'm not sure "multisig" does this full justice. How about say that "it is a pair of atomic payments: one from payer to intermediary and one from intermediary to payee"?